src/steps/generic.rs

```diff
@@ -101,6 +102,21 @@ pub fn run_vscode(run_type: RunType) -> Result<()> {
     Ok(())
 }
 
+pub fn run_micro(run_type: RunType) -> Result<()> {
+    let micro = utils::require("micro")?;
+
+    print_separator("micro");
+
+    let stdout = run_type.execute(&micro).args(&["-plugin", "update"]).string_output()?;
+    std::io::stdout().write_all(&stdout.as_bytes())?;
```

Comment:
Do we have to parse stdout? Doesn't micro set an nonzero exit code on failure?

Reply:
Unfortunately not, unless it crashes hard (thing segfault etc) it's always zero. https://github.com/zyedidia/micro/issues/1647

Reply:
Sorry I missed that